README.md

```diff
@@ -345,7 +345,7 @@ color themes.
 
     **Linux**
     * `~/.<PRODUCT><VERSION>/colors` Generic path,
-    * `~/.IntelliJIdeaXX/colors` [(IntelliJ IDEA)][IntelliJ IDE settings],
+    * `~/.IdeaICXX/config/colors` [(IntelliJ IDEA)][IntelliJ IDE settings],
```

Comment:
That is folder for Intellij Community Edition. The folder for regular IntellijJ IDEA might be the same as was before.